__dirname (or import.meta.dirname) is not available in this context if node is
run with --experimental-strip-types. This file is generated by wasm-pack, so we
must apply this patch after running wasm-pack.

This main reason for AMARO_WASM_BG_PATH env variable is for running tests.

--- a/pkg/wasm.js
+++ b/pkg/wasm.js
@@ -547,7 +547,7 @@
     throw new Error(getStringFromWasm0(arg0, arg1));
 };
 
-const path = require('path').join(__dirname, 'wasm_bg.wasm');
+const path = process.env.AMARO_WASM_BG_PATH || '/usr/lib/node_modules/amaro/dist/wasm_bg.wasm';
 const bytes = require('fs').readFileSync(path);
 
 const wasmModule = new WebAssembly.Module(bytes);
